internal/runbits/buildscript/buildscript.go

```diff
@@ -1,4 +1,4 @@
-package buildscript
+package buildscript_runbit
```

Comment:
Went ahead and named this according to what we want to start naming these based on our recent discussion on duplicate package naming.